nevergrad/benchmark/experiments.py

```diff
@@ -30,6 +30,63 @@
 # fmt: off
 
 
+@registry.register
+def yawidebbob(seed: Optional[int] = None) -> Iterator[Experiment]:
+    seedg = create_seed_generator(seed)
+    # Continuous case
+    # First, a few functions with constraints.
+    functions = [
+        ArtificialFunction(name, block_dimension=50, rotation=rotation) for name in ["cigar", "ellipsoid"] for rotation in [True, False]
+    ]
+    for func in functions:
+        func.parametrization.register_cheap_constraint(_positive_sum)
+    # Then, let us build a constraint-free case. We include the noisy case.
+    names = ["hm", "rastrigin"]  #, "griewank", "rosenbrock", "ackley", "lunacek", "deceptivemultimodal", "bucherastrigin", "multipeak"]
```

Comment:
can you do just one list and remove the comments? or is there some meaning there?

Reply:
Ok!